### ISSUE TITLE
[fix] a11y addon hung on "Initializing..."

### ISSUE THREAD

**Issue by Shkeating** [trussworks/react-uswds#2191]
**ReactUSWDS Version & USWDS Version:**

Most current version of this repo on the `main` branch

**Describe the bug**

**To Reproduce**
```
git checkout main
git pull
yarn install
yarn storybook
```

1. Go to 'localhost:9009'
2. Click on the `Accessibility` tab at the bottom of the page
3. See `Initializing...` message. It does not go away.

**Expected behavior**

I expect to see if there are any automated accessibility tests that failed, and if not, I want to see which ones ran on the component that passed.

**Screenshots**
<img width="1522" alt="image" src="https://user-images.githubusercontent.com/59394696/191333264-4965b883-9013-48cc-b10f-f69414c71eac.png">

**Additional context**

Addressed in this issue: https://github.com/storybookjs/storybook/discussions/18437 

It says you need to update the Storybook version to remediate the issue.

**Device and Browser Information (please complete the following information if describing a UI bug):**

N/A it happens on all devices I've checked on.
